src/examples/iris/helper.cpp

```diff
@@ -26,10 +26,10 @@ void readIrisData(const std::string fileName,
   }
   std::string line;
   std::string value;
-  while(marian::utils::getline(in, line)) {
+  while(marian::io::getline(in, line)) {
```

Comment:
This will not compile as `in` is `std::ifstream`. I think using `std::getline` should be safe here as the Iris data file is provided and it doesn't contain CRs. I'll fix this.